### ISSUE TITLE
Optimising body joints with camera_translation gradients enabled

### ISSUE THREAD

**Issue by nefeliandreou** [wangsen1312/joints2smpl#2]
Hi, thanks for sharing this project. I have a question regarding the part where the body joints are optimised. In [this line](https://github.com/wangsen1312/joints2smpl/blob/afe29d4ca59c3743a8057eb54344e9f422ac9c2a/src/smplify.py#L201), shouldn't the gradients for camera_translation be disabled, i.e. replace
```
camera_translation.requires_grad = True
```
by
```
camera_translation.requires_grad = False
```
or am I missing something? Thanks!

**Comment by wangsen1312** [comment]
@nefeliandreou Thank you for your point out. I have tried this two diffrerent settings, I found even in the second round make the camera_translation work will increase a little accuaracy so I left it here.
